scripts/integration_tests.ps1

```diff
@@ -0,0 +1,9 @@
+ECHO "-- Installing Golang"
+hab pkg install -b afiune/go
```

Comment:
If we merge this PR I will open an Issue to do the switch to the `core/go` package once it gets merged! 💯